packages/ui/Cancelable/tests/Cancelable-specs.js

```diff
@@ -351,5 +351,31 @@ describe('Cancelable', () => {
 				expect(actual).toBe(expected);
 			}
 		);
+
+		test('should invoke modal handlers in LIFO order', () => {
```

Comment:
Let's rename the title so that there's no duplicate.

Reply:
And also let's move it up and group with the other mounting order test.

Reply:
Whoops. Yeah that makes sense